calculator.go

```diff
@@ -27,15 +27,15 @@ func Multiply(a, b float64) float64 {
 // from the first.
 func Divide(a, b float64) (float64, error) {
 	if b == 0 {
-		return 0, errors.New("division by zero error")
+		return 0, fmt.Errorf("invalid inputs(%f, %f): division by zero error", a, b)
 	}
 	return a / b, nil
 }
 
 // Sqrt takes one numbers and returns the square root
 func Sqrt(a float64) (float64, error) {
 	if a < 0 {
-		return 0, errors.New("division by zero error")
+		return 0, fmt.Errorf("invalid input(%f): division by zero error", a)
```

Comment:
Not actually a division by zero error; it's just that there _is_ no natural number whose square is negative, so asking for the square root of a negative number is meaningless on the face of it.

Reply:
nice one, copy paste error :-) fixed